GherkinSyncTool.Synchronizers.AllureTestOps/Client/AllureClient.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using GherkinSyncTool.Models;
+using GherkinSyncTool.Models.Configuration;
+using GherkinSyncTool.Synchronizers.AllureTestOps.Exception;
+using GherkinSyncTool.Synchronizers.AllureTestOps.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using NLog;
+using Quantori.AllureTestOpsClient;
+using Quantori.AllureTestOpsClient.Model;
+using Refit;
+
+namespace GherkinSyncTool.Synchronizers.AllureTestOps.Client
+{
+    public class AllureClient
+    {
+        private static readonly Logger Log = LogManager.GetLogger(MethodBase.GetCurrentMethod()?.DeclaringType?.Name);
+
+        private readonly AllureTestOpsSettings _azureDevopsSettings =
+            ConfigurationManager.GetConfiguration<AllureTestOpsConfigs>().AllureTestOpsSettings;
+
+        private readonly Context _context;
+        private readonly IAllureClient _allureClient;
+
+        public AllureClient(Context context)
+        {
+            _context = context;
+            _allureClient = RestService.For<IAllureClient>(_azureDevopsSettings.BaseUrl, new RefitSettings
+            {
+                AuthorizationHeaderValueGetter = () => Task.FromResult(_azureDevopsSettings.AccessToken),
+                ContentSerializer = new NewtonsoftJsonContentSerializer(
+                    new JsonSerializerSettings
+                    {
+                        ContractResolver = new CamelCasePropertyNamesContractResolver(),
+                        NullValueHandling = NullValueHandling.Ignore
+                    }
+                )
+                
+            });
+        }
+
+        public IEnumerable<Quantori.AllureTestOpsClient.Model.Content> GetAllTestCases()
+        {
+            var allContent = new List<Quantori.AllureTestOpsClient.Model.Content>();
+            var isLastElementOnThePag = false;
```

Comment:
'e' is missed in the end of variable name

Reply:
Fixed